client/src/main/proto/spine/client/query.proto

```diff
@@ -46,30 +47,75 @@ message QueryId {
     string value = 1;
 }
 
+// An order for Query results.
+//
+message Order {
+    
+    // A direction of sorting for ordering the Query results.
+    //
+    Direction direction = 1 [(required) = true];
+
+    // A column name which value is used for ordering the Query results. 
+    //
+    string column = 2;
+    
+    enum Direction {
+        OD_UNKNOWN = 0;
+        ASCENDING = 1;
+        DESCENDING = 2;
+    }
+}
+
+// Pagination parameters for Query results.
+//
+message Pagination {
+
+    // An amount of entities to be returned by executing a Query.
+    //
+    // A minimum of 1 items per page can be requested
+    //
+    uint64 page_size = 1 [(required) = true];
+}
+
 // Allows clients to form the requests to the read-side through the `QueryService`.
 //
 // `Query` execution typically results in a `QueryResponse` object.
 //
 message Query {
 
     // The ID of the query.
+    //
     QueryId id = 1 [(required) = true];
 
     // Defines the entity of interest, e.g. entity type URL and a set of fetch criteria.
+    //
     Target target = 2 [(required) = true,
                        (valid) = true,
                        (if_invalid).msg_format = "Invalid query target"];
 
     // Field mask to be applied to the items of the query result.
+    //
     google.protobuf.FieldMask field_mask = 3;
 
     // Service information about the environment in which the query was created.
+    //
     core.ActorContext context = 4 [(required) = true,
                                    (valid) = true,
                                    (if_invalid).msg_format = "Invalid actor context"];
 
+    // An order for entities returned by the Query.
+    //
+    Order order = 5 [(valid) = true,
+                     (if_invalid).msg_format = "Invalid pagination order"];
```

Comment:
That's just an order. It's no "pagination order". Users can set just the order, without pagination.